tests/framework/e2e/etcdctl.go

```diff
@@ -243,3 +249,24 @@ func (ctl *EtcdctlV3) Health() error {
 	return SpawnWithExpects(args, map[string]string{}, lines...)
 
 }
+
+func (ctl *EtcdctlV3) Defragment(o config.DefragOption) error {
+
+	args := append([]string{}, "defrag")
+	if o.Timeout != 0 {
+		args = append(args, fmt.Sprintf("--command-timeout=%s", o.Timeout))
+	}
+	if o.WithUtl {
+		args = append(ctl.amdArgsWithUtl(), args...)
+		args = append(args, "--data-dir", ctl.runningCfg[0].DataDirPath)
+		lines := []string{"finished defragmenting directory"}
+		return SpawnWithExpects(args, map[string]string{}, lines...)
+	}
+	args = append(ctl.cmdArgs(), args...)
+	lines := make([]string, len(ctl.endpoints))
+	for i := range lines {
+		lines[i] = "Finished defragmenting etcd member"
```

Comment:
Based on this I surmise that etcdctl will send defrag request to each etcd member?

Reply:
It will send to etcd member with `--endpoint`. 

> Based on this I surmise that etcdctl will send defrag request to each etcd member?


In current. Yes